browser/ui/views/sidebar/sidebar_container_view.h

```diff
@@ -107,11 +119,19 @@ class SidebarContainerView
 
   void AddChildViews();
   void UpdateBackground();
-  void UpdateSidebarVisibility();
-  void UpdateSidebarVisibility(
-      sidebar::SidebarService::ShowSidebarOption show_option);
   void ShowOptionsEventDetectWidget(bool show);
-  void ShowSidebar(bool show_sidebar, bool show_event_detect_widget);
+  bool ShouldUseAnimation() const;
+
+  // Show control view. panel's visibility depends on |show_side_panel|.
+  void ShowSidebar(bool show_side_panel);
+
+  // Hide panel. Control view's visibility depends on |hide_sidebar_control|.
+  void HideSidebar(bool hide_sidebbar_control);
```

Comment:
totally optional: It could be just me, but I always need to come back here when runnning into a callsite with `true/false`. Can we have a base::StrongAlias() or option enums for readability?

Reply:
Added helper methods for readability.
```
ShowSidebarAll();
ShowSidebarControlView();
HideSidebarAll();
HideSidebarPanel();
```